README.md

```diff
@@ -1,16 +1,43 @@
+With the command `python vim_menutrans_scanner.py <vimruntime> menu_trans.vim`
+`.template ` under project root dictionay, you could get the `Untranslated `
+`messages` list and `Deleted messages` list, which tells you the messages need 
+to be translated and messages need to be removed from the temple.
 
-* `vim_menutrans_merger.py` is to apply `menu_trans.vim.template` to every language. With the command ```python vim_menutrans_merger.py menu_trans.vim.template /usr/share/vim/vim82/runtime/translationfile ``` under project root dictionary, you could generate the menu items template in `translationfile`. Since `vim_menutrans_merger.py` will cover the `translationfile`, it's not recommended to apply the command directly to the installed version on the system. You should apply it to the translation files in Source Tree, or create a copy and apply it to the copy if your want to revise the file on System. Untranslated items in the generated files will be annotated. The annotations should be removed after the translations are completed. There are three translation formats: `"TRANSLATION MISSING", TRANSLATION\ MISSING, TRANSLATION MISSING`. `"TRANSLATION MISSING"` shouldn't be dropped the quotes. `TRANSLATION\ MISING` shouldn't be dropped the backslash, which excapes the blank behind it and makes "TRANSLATION" and "MISING" stick together.
+* `vim_menutrans_merger.py` is to apply `menu_trans.vim.template` to every 
+language. With the command `python vim_menutrans_merger.py menu_trans.vim.`
+`template /usr/share/vim/vim82/runtime/translationfile` under project root 
+dictionary, you could generate the menu items template in `translationfile`. 
+Since `vim_menutrans_merger.py` will cover the `translationfile`, it's not 
+recommended to apply the command directly to the installed version on the 
+system. You should apply it to the translation files in Source Tree, or create 
+a copy and apply it to the copy if your want to revise the file on System. 
+Untranslated items in the generated files will be annotated. The annotations 
+should be removed after the translations are completed. There are three 
+translation formats: `"TRANSLATION MISSING", TRANSLATION\ MISSING, TRANSLATION `
+`MISSING`. `"TRANSLATION MISSING"` shouldn't be dropped the quotes. 
+`TRANSLATION\ MISSING` shouldn't be dropped the backslash, which excapes the 
+blank behind it and makes "TRANSLATION" and "MISSING" stick together.
 
 * `<vimruntime>`   
-    + If you haven't installed Vim, just cloned the Vim official code repository, it's path is `projectroot/runtime/`.
-    + If you have installed Vim on Windows, the default path is `C:\Program Files\Vim\vim82`.
-    + If you have installed Vim on Linux, the default path is `/usr/share/vim/vim82`.
+    + If you haven't installed Vim, just cloned the Vim official code 
+    repository, it's path is `projectroot/runtime/`.
+    + If you have installed Vim on Windows, the default path is `C:\Program `
+    `Files\Vim\vim82`.
```

Comment:
You should not wrap inside code snippet. Therefore, wrap the line before `C:\Program...`.

Also, don't forget to increase the indent level of the next line by 2 spaces.